### PR TITLE
Add self.body to actions generated with --skip-view flag

### PR DESCRIPTION
**Pull request by mvader**
Implemented the changes proposed in #203 after the merge.

### CODE REVIEW

**Comment by joneslee85** [comment]
:+1:

File: test/commands/generate_test.rb [hunk 1]
```diff
@@ -59,6 +59,7 @@ def default_options
           content.must_match %(  class Index)
           content.must_match %(    include Web::Action)
           content.must_match %(    def call(params))
+          content.wont_match %(      self.body = 'OK')
```
Comment (AlfonsoUceda) [inline test/commands/generate_test.rb]: mmm one thing, in this test `--skip-view` option isn't used, isn't it?

Comment (mvader) [inline test/commands/generate_test.rb]: @AlfonsoUceda nope, that's why I test that self.body is not present in the resultant action. Note the ```wont_match```

Comment (AlfonsoUceda) [inline test/commands/generate_test.rb]: Oh you are rigth, I didn't see 'wont_match' sorry >_<

**Pull request merged**